maven-javadoc-plugin/pom.xml

```diff
@@ -193,9 +193,9 @@ under the License.
 
     <!-- misc -->
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.6</version>
```

Comment:
3.6 added quite a lot of support for Java 9, do you know how well 3.6 works with Java 9?